### PR TITLE
[DPEDE-1683] Add prop to make column customization btn visible in too…

### PR DESCRIPTION
**Pull request by xft-dev**
[https://ctl.atlassian.net/browse/DPEDE-1683](https://ctl.atlassian.net/browse/DPEDE-1683)

### CODE REVIEW

File: src/chi-vue/src/components/data-table-toolbar/DataTableToolbar.vue [hunk 1]
```diff
@@ -1,5 +1,12 @@
 <script lang="ts" setup>
 import { DATA_TABLE_CLASSES } from '@/constants/classes';
+import ColumnCustomization from '../../components/column-customization/ColumnCustomization';
+import { DataTableColumnsData } from '@/constants/types';
+
+const props = defineProps<{
+  columnCustomizationBtn?: boolean;
+  columnsData?: DataTableColumnsData | {};
```
Comment (ASanchLumen) [inline src/chi-vue/src/components/data-table-toolbar/DataTableToolbar.vue:8]: We can avoid creating two props and check if there is columns data. Also, columnsData cannot be `{}` as it requires the key `columns`:

```suggestion
  columnCustomizationData?: DataTableColumnsData;
```

```vue
        <ColumnCustomization v-if="columnCustomizationData" :columnsData="columnCustomizationData" />
```

**Review by ASanchLumen — CHANGES_REQUESTED** [review]
Code works great! Besides the code comment to simplify, don't forget visual & functional tests!

**Comment by xft-dev** [comment]
[https://github.com/CenturyLink/Chi/pull/1153/commits/b3355c4949c83bd8ca12bc62e5f143b91a6bda63](https://github.com/CenturyLink/Chi/pull/1153/commits/b3355c4949c83bd8ca12bc62e5f143b91a6bda63)

Updated. 

Note: Typescript marks a type error in prop.

image

**Comment by xft-dev** [comment]
[https://github.com/CenturyLink/Chi/pull/1153/commits/948acb55a480e432a74b2a3ca488a0ec7fb7e930](https://github.com/CenturyLink/Chi/pull/1153/commits/948acb55a480e432a74b2a3ca488a0ec7fb7e930)

Tests added. Ready to review

**Comment by ASanchLumen** [comment]
> [b3355c4](https://github.com/CenturyLink/Chi/commit/b3355c4949c83bd8ca12bc62e5f143b91a6bda63)
> 
> @ASanchLumen Updated. Thanks.
> 
> Note: Typescript marks a type error in prop.
> 
> image

The error is caused because the fixtures have no types, it can be avoided by explicitly saying that it's the correct type:

image

**Review by ASanchLumen — APPROVED** [review]

**Pull request closed without merging**